modules/apps/commerce/commerce-product-content-web/src/main/resources/META-INF/resources/product_detail/render/view.jsp

```diff
@@ -205,6 +205,41 @@ long cpDefinitionId = cpCatalogEntry.getCPDefinitionId();
 					skuOptions="[]"
 				/>
 
+				<c:if test="<%= commerceContext.isRequestQuoteEnabled() %>">
+
+					<%
+					Group group = layout.getGroup();
+
+					String siteDefaultURL = HtmlUtil.escape(group.getDisplayURL(themeDisplay, layout.isPrivateLayout()));
+
+					CommerceCurrency commerceCurrency = commerceContext.getCommerceCurrency();
+					%>
+
+					<clay:button
+						additionalProps='<%=
+							HashMapBuilder.<String, Object>put(
+								"accountId", String.valueOf(commerceAccount.getCommerceAccountId())
+							).put(
+								"channelId", String.valueOf(commerceContext.getCommerceChannelId())
+							).put(
+								"currencyCode", commerceCurrency.getCode()
+							).put(
+								"quantity", 1
+							).put(
+								"siteDefaultURL", siteDefaultURL
+							).put(
+								"skuId", String.valueOf(cpSku.getCPInstanceId())
+							).build()
+						%>'
+						cssClass="btn-fixed btn-lg btn-secondary ml-3"
+						disabled="<%= commerceAccount == null %>"
+						displayType="secondary"
+						label='<%= LanguageUtil.get(request, "request-a-quote") %>'
+						propsTransformer="product_detail/render/js/RequestQuotePropsTransformer"
```

Comment:
This will be addressed in [COMMERCE-11324](https://issues.liferay.com/browse/COMMERCE-11324)